nikanorov-alex/Pr4/WeatherFather.py

```diff
@@ -0,0 +1,59 @@
+from bs4 import BeautifulSoup
+class WeatherFather():
+    def __init__(self, html, mainClass, mainNextDay, tabsContentInner, temperature, pClassOne, pClassTwo, pClassThree, min, max):
+        self.html = html
+        self.mainClass = mainClass
+        self.mainNextDay = mainNextDay
+        self.tabsContentInner = tabsContentInner
+        self.temperature = temperature
+        self.pClassOne = pClassOne
+        self.pClassTwo = pClassTwo
+        self.pClassThree = pClassThree
+        self.min = min
+        self.max = max
+    def start_gettingSinoptik(self):
+        soup = BeautifulSoup(self.html, 'html.parser')
+        # Ищет в этом классе
+        items = soup.find_all('div', class_=self.mainClass)
+        weather1 = []
+        weather2 = []
+        weather3 = []
```

Comment:
What the `weather1` stands for? Please name it so it reflects the actual data it conveys.